src/repositories/ColorRepository.ts

```diff
@@ -44,7 +43,7 @@ export const deleteColor = async (colorId: number): Promise<void> => {
     }
 };
 
-export function mapColorApiToFrontifyColor(color: ColorApi): FrontifyColor {
+export function mapColorApiToFrontifyColor(color: ColorApi): Color {
```

Comment:
Should update method name as well:
```suggestion
export function mapColorApiToColor(color: ColorApi): Color {
```